.drone.yml

```diff
@@ -85,6 +85,24 @@ pipeline:
       matrix:
         TEST_SUITE: phpunit
 
+  phan:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - make test-php-phan
+    when:
+      matrix:
+        TEST_SUITE: phan
+
+  phpstan:
```

Comment:
there is a ``phpstan`` section in ``services`` - please investigate, probably delete it

Reply:
I'll remove that. They never worked